frank-flow/src/frontend/src/assets/styles/jqx.base.scss

```diff
@@ -45,7 +45,7 @@
   background: var(--bg-color-muted);
 
   &.jqx-tree-item {
-    background: none;
+    //background: none;
```

Comment:
comment

Reply:
Solved this, by using a different color.